SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+if you believe you have found a security vulnerability in our repository, please report it to us as described below.
+
+## Reporting a Vulnerability
+
+To report a vulnerability, send and email to the adress below : [dsi-oscoe-enedis@enedis.fr](mailto:dsi-oscoe-enedis@enedis.fr)
+
+Please report security vulnerabilities by providing information listed below :
+- Project Name (url to the project repository)
```

Comment:
ça n'est pas forcément évident de comprendre pourquoi dans la mesure où ce fichier est présent dans 1 projet.
Peut-être préciser que cette adresse est générique pour l'ensemble des projets OSS Enedis et que pour nous aider on a besoin de catégoriser un peu.

ex. 
- Project name `[mocha-xray-reporter]`
  - In order to help us know the project since our email address is generic for all our projects